crates/rpc/src/handlers/config.rs

```diff
@@ -61,7 +44,34 @@ impl From<Arc<NetworkSpec>> for SpecConfig {
 }
 
 /// Called by `config/spec` to get specification configuration.
-pub async fn get_spec(network_spec: Arc<NetworkSpec>) -> Result<impl Reply, Rejection> {
+#[get("config/spec")]
+pub async fn get_config_spec(
+    network_spec: web::Data<Arc<NetworkSpec>>,
+) -> actix_web::Result<impl Responder> {
+    let network_spec = network_spec.as_ref();
+    let spec_config = SpecConfig::from(network_spec);
+
+    Ok(HttpResponse::Ok().json(spec_config))
+}
+#[get("config/spec1")]
+pub async fn get_config_spec1(
+    network_spec: web::Data<Arc<NetworkSpec>>,
+) -> actix_web::Result<impl Responder> {
+    let network_spec = network_spec.as_ref();
```

Comment:
`#[get("config/spec")]` `#[get("config/spec1")]` why do we have the same thing twice, I don't think `#[get("config/spec1")]` is an endpoint